### PR TITLE
add documentation section for common View methods

### PR DESCRIPTION
**Pull request by NickIliev**
Related to [https://github.com/NativeScript/docs/issues/995](https://github.com/NativeScript/docs/issues/995)

### CODE REVIEW

File: docs/ui/components.md [hunk 1]
```diff
@@ -34,13 +34,15 @@ NativeScript ships with a set of user interface [`views`](http://docs.nativescri
...
-> **TIP:** You can access the underlying native widget for each view at runtime using the following properties:
+> **TIP:** You can access the underlying native widget for each view at runtime using the `nativeView` property:
 >
-> * Android: `<view>.android`
-> * iOS: `<view>.ios`
+>  `<view>.nativeView`
 >
 > Accessing the native widgets might be useful when you want to use some platform-specific functionalities of the widget. You can find information about the underlying native component for each view below.
 
+Each component comes with a set of common methods availabel from the extended `View` class.
```
Comment (etabakov) [inline docs/ui/components.md]: A typo in "available"

File: docs/ui/components.md [hunk 2]
```diff
@@ -34,13 +34,15 @@ NativeScript ships with a set of user interface [`views`](http://docs.nativescri
...
+Dedicated documentatio0n section covering the different measuring and position methods can be found [here]({%slug properties#view-class-common-methods %).
```
Comment (etabakov) [inline docs/ui/components.md]: A typo in "documentation"

**Review by etabakov — COMMENTED** [review]
Looks good. Just 2 typos found.

**Comment by etabakov** [comment]
Also, there is a problem in the markdown => the build fails: http://nsbuild01.telerik.com/build/job/docs-stable-PR/44/console

**Comment by NickIliev** [comment]
Build fixed via [https://github.com/NativeScript/docs/pull/1056/commits/df14625ed44f98f4f2e5070612931b3fb66049c6](https://github.com/NativeScript/docs/pull/1056/commits/df14625ed44f98f4f2e5070612931b3fb66049c6)

**Pull request merged**